src/components/seeds/database/TableCellRenderer.tsx

```diff
@@ -52,15 +52,19 @@ export default function SearchCellRenderer(props: RendererProps<SearchResponseEl
         index={index}
         column={column}
         value={
-          <Link fontSize='16px' to={APP_PATHS.ACCESSIONS2_ITEM.replace(':accessionId', `${row.id}`)}>{`${value}`}</Link>
+          <Link
+            fontSize='16px'
+            to={APP_PATHS.ACCESSIONS2_ITEM.replace(':accessionId', `${row.id as string | number}`)}
+          >{`${value as string}`}</Link>
         }
         row={row}
       />
     );
   }
 
   const numberCell = (units: unknown) => (
-    <CellRenderer index={index} column={column} value={`${value} ${units}`} row={row} />
+    // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
+    <CellRenderer index={index} column={column} value={`${value as number} ${units}`} row={row} />
```

Comment:
Could it be units as strings instead of disabling?

Reply:
I couldn't tell if `units` was likely to be anything else. If we think it should always be a string going into `numberCell`, I'm good to update to `units as string`